packages/fast-components-react-msft/src/action-toggle/action-toggle.props.ts

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import { Omit } from "utility-types";
+import {
+    ActionToggleClassNameContract,
+    ManagedClasses,
+} from "@microsoft/fast-components-class-name-contracts-msft";
+import {
+    ButtonAppearance,
+    ButtonHandledProps,
+    ButtonManagedClasses,
+    ButtonUnhandledProps,
+} from "../button/button.props";
+
+export enum ActionToggleAppearance {
+    lightweight = ButtonAppearance.lightweight,
+    primary = ButtonAppearance.primary,
+}
+
+export interface ActionToggleManagedClasses
+    extends ManagedClasses<ActionToggleClassNameContract> {}
+export interface ActionToggleHandledProps
+    extends Omit<ButtonHandledProps, "appearance" | keyof ButtonManagedClasses>,
+        ActionToggleManagedClasses {
+    /**
+     * The action toggle selected glyph render prop
```

Comment:
Since these are actually more about a UI state, I would rewrite the comment to something more like:
"The action toggle selected state glyph"
Same for the unselectedGlyph